pkg/sfu/receiver.go

```diff
@@ -298,7 +298,12 @@ func (w *WebRTCReceiver) AddUpTrack(track *webrtc.TrackRemote, buff *buffer.Buff
 	}
 
 	layer := int32(0)
-	if w.Kind() == webrtc.RTPCodecTypeVideo {
+	// for svc codecs, use layer full quality instead.
+	// we only have buffer for full quality
+	if w.isSVC {
+		layer = int32(len(w.buffers)) - 1
+	}
```

Comment:
This is the part that is puzzling. If I consistently use `layer = 0` for SVC codecs, the PLI layer lock keeps spamming (video & screen share is received at the other end, but adaptive stream/dynacast does not seem to work properly with the received resolution jumping around). Using `layer = 0` does not cause that behaviour in v1.3.0 of server.

That points to opportunistic forwarding probably jumping around with layers (allowing overshoot) etc. not working well. But, I have to dig to figure out why using `layer = 2` does not cause PLI layer lock spamming, but `layer = 0` does.

Reply:
It looks good for js-sdk implementation that always publish L3T3 for svc now, but the proper way is making js-sdk to publish the correct svc layers in TrackInfo, then we can remove this line here.